samples/snippets/load_table_schema_from_json.py

```diff
@@ -0,0 +1,60 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pathlib
+
+
+def load_table(table_id: str):
+    orig_uri = "gs://cloud-samples-data/bigquery/us-states/us-states.csv"
+    orig_table_id = table_id
+    current_directory = pathlib.Path(__file__).parent
+    orig_schema_path = current_directory / "schema_us_states.json"
```

Comment:
I think we need a string here as well.

```suggestion
    orig_schema_path = str(current_directory / "schema_us_states.json")
```